packages/components/src/color-picker/style.native.scss

```diff
@@ -42,6 +42,20 @@
 	color: $dark-primary;
 }
 
+.selectColorText {
+	text-align: center;
+	color: $gray;
+	font-size: 16px;
+	line-height: 24px;
+}
+
+.selectColorTextDark {
+	text-align: center;
+	color: $dark-primary;
+	font-size: 16px;
+	line-height: 24px;
+}
+
```

Comment:
Since we are concatenating these styles using  `usePreferredColorSchemeStyle` we could only set the color for `selectColorTextDark` and leaving the rest of the attributes in the `selectColorText` style. This way it'd use those as defaults and only replace the color if the dark mode theme is turned on.

Reply:
Your suggestion looks good :)
I've submitted the change.